pages/coinbase/signin.py

```diff
@@ -53,7 +53,7 @@ def build_elements(self):
         # NAVIGATION BAR
 
         self.home_link =      Link(self.driver, self.home_link_loc)
-        self.products_menu =  ReactiveMenu(self.driver, self.products_menu_loc)
+        self.products_menu =  ReactiveMenu(self.driver, self.products_menu_loc) #needs a driver.maximize_window() before instantiation to successfully implement this object and the remaining objects in the navigation bar
```

Comment:
I know. It doesn't need a maximize window though because it's a resolution issue - if your monitor is small enough maxing the window won't help. We just need to make sure the window is a certain size. But yeah, almost the same thing.

Anyway, please write a bug ticket for this problem.